tdrs-backend/tdpservice/parsers/tanf_parser.py

```diff
@@ -43,15 +35,25 @@ def active_t1_parser(line, line_number):
         # logger.debug('field: %s\t::content: "%s"\t::end: %s', field.name, content, field.end)
 
         if content_is_valid:
-            setattr(t1, field.name, content)
+            setattr(model, field.name, content)
 
-    if content_is_valid is False:
+    if not content_is_valid:
         logger.warn('Content is not valid, skipping model creation.')
         return
 
     # try:
-    # t1.full_clean()
-    t1.save()
+    # model.full_clean()
+    model.save()
+
+def active_t1_parser(line, line_number):
+    """Parse line in datafile as active case data, T1 only."""
+    family_case_schema = t1_schema()
+    # create search_index model
+    t1 = T1()
+
+    min_line_length = 118  # we will need to adjust for other types
```

Comment:
Add a constant file and define 118 in that file, something like: MIN_PARSE_LINE_LENGTH = 118